app/routes/orders/view.js

```diff
@@ -11,10 +11,13 @@ export default Route.extend({
       include : 'attendees,tickets,event',
       reload  : true
     });
-    const eventDetails = await order.query('event', {});
+    const eventDetails = await order.query('event', {
+      include: 'tax'
+    });
     return {
       order,
-      form: await eventDetails.query('customForms', {
+      event : eventDetails,
+      form  : await eventDetails.query('customForms', {
         'page[size]' : 50,
```

Comment:
@mrsaicharan1 I am using : 
```js

const customEventDetails = await order.query('event', {include : 'tax'});
    return {
      order,
      event : customEventDetails,
      form: await eventDetails.query('customForms', {
        'page[size]' : 50,
        sort         : 'id'
      })
    };
```
see if it works for you if it hasn't been for now

Reply:
Yes, believe that creating a separate `customEventDetails` is more appropriate.